components/components/navigation/menu-main/menu-main.css

```diff
@@ -0,0 +1,87 @@
+/**
+ * @file
+ * This file is used to style the main menu.
+ */
+
+/*
+  Main Menu
+
+  This section presents the Main Menu of the website.
+
+  Markup: menu-main.html.twig
+
+  Style guide: components.navigation.main_menu
+*/
+
+.menu-main__toggle {
+  width: 40px;
+  height: 40px;
+  line-height: 1;
+  padding: 0;
+  border: none;
+  text-align: left;
+  background-color: transparent;
+}
+.menu-main {
+  margin: 0;
+  padding: 0;
+  max-height: 0;
+  overflow: hidden;
+  list-style-type: none;
+  font-family: 'Scope One';
+  font-weight: normal;
+  font-size: 1.266rem;
+  line-height: 1.2;
+  color: #000;
+  transition: max-height 0.5s ease-in;
+}
+.menu-main.is-active {
+  /**
+   * An exact max-height value is must be set to make possible to animate the menu display.
+   * It my not be much larger than the content because that would break the animation.
+   * See: https://css-tricks.com/using-css-transitions-auto-dimensions/
+   */
+  max-height: 18.75rem;
+  overflow-y: auto;
+}
+.menu-main__item {
+  margin-top: 0.8em;
+}
+.menu-main__link {
+  display: block;
+  padding-bottom: 0.15em;
+  text-decoration: none;
+  border-bottom: solid 0.15em transparent;
+  color: inherit;
+  transition: all 0.2s;
+}
+.menu-main__link:hover,
+.menu-main__link.is-active:hover,
+.menu-main__link:focus {
+  text-decoration: none;
+  border-bottom-color: rgba(255, 97, 56, 0.3);
+  color: #ff6138;
+}
+.menu-main__link:active,
+.menu-main__link.is-active {
+  text-decoration: none;
+  border-bottom-color: #ff6138;
+}
+
+@media screen and (min-width: 30em) {
```

Comment:
Would it be possible to add a variation of the responsive navigation so that this could be shown on the desktop?

Reply:
@lauriii I'm afraid I do not understand this request… :|

Reply:
@lauriii Oh, you mean to see the narrow menu on desktop? Weel I could add a narrow container or a class to make the nav narrow but it would be an unneeded thing in the theme. 

I do not know if it is possible to manage it just in the styleguide to have a narrow container just in that without affecting the css added to the theme…